assets/css/global.css

```diff
@@ -70,7 +70,7 @@
 
     --button-background: var(--polar-night-light);
     --button-text: var(--snow-storm-light);
-    --button-primary-background: var(--frost-dark);
+    --button-primary-background: var(--primary-color-light);
```

Comment:
I don't see why this should be changed. The green is a bit too vibrant and doesn't fit well with the blueish theme.
![image](https://user-images.githubusercontent.com/60521810/152632532-bea83498-cda9-48a7-bd3c-f86c3b995012.png)

Reply:
alright.
i thought that a blue-and-green colour scheme would go better

Reply:
@AnAlgorithm Maybe you are right. However, I'm currently in the process of reworking the structure of the CSS variables (and how they are used) to simplify themes and achieve better consistency. Therefore, I think we should put this change on hold until I finalize the CSS variables.

Reply:
okay